src/clang.rs

```diff
@@ -149,72 +206,98 @@ impl Cursor {
             parent.is_in_non_fully_specialized_template()
     }
 
+    /// Is this cursor pointing a valid referent? In other words, is it *not*
+    /// the null cursor?
```

Comment:
Hm... I'm not sure this is always true, to be fair. What'd be the point of having `clang_isInvalid` and `clang_isNullCursor`? If it holds, we can pretty much remove one of them.